platforms/common/uORB/uORBDeviceNode.cpp

```diff
@@ -925,3 +934,17 @@ uORB::DeviceNode::_unregister_callback(uorb_cb_handle_t &cb_handle)
 
 	return ret;
 }
+
+#ifndef CONFIG_BUILD_FLAT
+void
+uORB::DeviceNode::set_max_writer_priority()
+{
+	sched_param param;
```

Comment:
There is a syscall like sched_getparam() which works for all thread types (task / kthread / pthread), mabye use that instead ?

Reply:
Very good point, will do. It is also much cleaner as it doesn't point to "pthread" directly

Reply:
done

Reply:
done